packages/app/.env.test

```
@@ -1,4 +1,5 @@
 VITE_FUEL_PROVIDER_URL=http://localhost:4001/graphql
 VITE_FUEL_FAUCET_URL=http://localhost:4041/dispense
-VITE_CONTRACT_ID=0x7e35610f31ce8f60b8081b8883db920f6169535020e83904d23e7ae35801d9c4
-VITE_TOKEN_ID=0x5490ccd5425acac521924bd537e4a26b2124427f736a7e56da08c486dc676a25
+VITE_FAUCET_RECAPTCHA_KEY=
+VITE_CONTRACT_ID=0x1491d2949e22d7d28e418f7b4c33bf75e3378ece70be585b960b225a6c1f3182
+VITE_TOKEN_ID=0xf7a2eb8a271a3ea73d6fb8756e272f4a1687d9522acf2ded4454f5d0d5f6a660
```

The `swayswap-scripts` will automatically set this envs, I think we can zero this;

```suggestion
VITE_CONTRACT_ID=0x0000000000000000000000000000000000000000000000000000000000000000
VITE_TOKEN_ID=0x0000000000000000000000000000000000000000000000000000000000000000
```